cardinal/server/server.go

```diff
@@ -355,6 +358,78 @@ func registerReadHandlerSwagger(world *ecs.World, api *untyped.API, handler *Han
 		getListTxReceiptsReplyFromRequest(world),
 	)
 
+	cqlHandler := runtime.OperationHandlerFunc(func(params interface{}) (interface{}, error) {
```

Comment:
optional: `any` can be used as an alias to `interface{}`. makes it a tiny bit less verbose, but up to you if you wanna use it!

Reply:
eh. Next time.